### DISCUSSION TITLE
Add `bail` flag for check command

### DISCUSSION THREAD

**Discussion by jtuchel** [biomejs/biome#3938] [Ideas]
Our project can be tested, formatted etc. locally, during CI the pipeline should fail as fast as possible. It would be nice to have a `--bail` flag for `biome check` so it fails on the first error.

Example docs:

```
bail

    --bail <number>

Stop execution when given number of checks have failed (default: 0)
```

**Comment by ematipico** [comment]
That's a good suggestion. I think this option should fit the `biome ci` command instead. You don't want to use `check` in CI

**Reply by jtuchel** [reply]
Cool :)

Created a new issue => https://github.com/biomejs/biome/issues/3992
